### DISCUSSION TITLE
Uninstall Macs using Uninstall-FalconSensor

### DISCUSSION THREAD

**Discussion by Sl1m007** [CrowdStrike/psfalcon#293] [Q&A] [answered]
Hi there,

When would the 'Uninstall-FalconSensor' command be available for Macs, please?

Thank you in advance.

**Comment by bk-cs** [comment, marked as answer]
I'll put in on my list for the next PSFalcon release (v2.2.5).

**Reply by mmaticnav** [reply]
has this been resolved since being posted? I am running 2.2.6 and still getting an error :(

**Reply by bk-cs** [reply]
No, it hasn't yet. I've got a pretty long list and must have forgot to get to it. I'll add a note to an [the existing Add-FalconSensorTag issue](https://github.com/CrowdStrike/psfalcon/issues/400) to better track it.
